minard/polling.py

```diff
@@ -256,10 +431,15 @@ def channel_information(conn):
 
     return channel_info
 
-def relay_status(conn):
+
+def relay_status(conn, run):
+    """
+    Returns the hv relay masks
+    """
```

Comment:
This should use a dictionary since all the crate states are not guaranteed to be in there.